app/models/form/subsection.rb

```diff
@@ -30,7 +30,7 @@ def status(log)
 
     qs = applicable_questions(log)
     qs_optional_removed = qs.reject { |q| log.optional_fields.include?(q.id) }
-    return :not_started if qs.count.positive? && qs.all? { |question| log[question.id].blank? || question.read_only? || question.derived? }
+    return :not_started if qs.count.positive? && qs.all? { |question| question.unanswered?(log) || question.read_only? || question.derived? }
```

Comment:
this is to make the status show correctly as checkbox questions with multiple boxes have different ids for each checkbox rather than using the question_id

Reply:
It seems to be working correct for the lettings logs checkbox questions. Could we make it work the same way?

Reply:
Ah - I think it only looks like it's working for lettings logs because there are other questions in those subsections - `:not-started` won't be triggered incorrectly in that case because it won't pass the qs.all? (even though the log[question.id] will be incorrectly blank when individual checkbox ids are filled in)

Reply:
There's an argument that it doesn't matter because we're always going to be adding other questions to the section but I think it's still good to fix it (can't think of a case it matters but there may be one)

Reply:
Okay, that's fine. I was worried we're looping a lot, but we don't really have that many checkbox questions. It is actually a nice change, well done!